modules/core/src/smithy4s/StaticService.scala

```diff
@@ -0,0 +1,37 @@
+package smithy4s
+
+import smithy4s.kinds.FunctorK5
+import smithy4s.kinds.PolyFunction5
+
+trait StaticService[P[_[_, _, _, _, _]]] extends FunctorK5[P] {
+  type Alg[_[_, _, _, _, _]]
+  val service: Service[Alg]
+  def endpoints: P[service.Endpoint]
```

Comment:
```suggestion
  def endpointsProduct: Prod[service.Endpoint]
```

Reply:
done as part of c5e96829819f7d69180862005bad057dcde0083b